### PR TITLE
Add clear button on multiselect example

### LINKED ISSUE #2309: MultiSelect should support clear all selected tags
I think MultiSelect should have a clear button and so you can click it to clear selected items. 
Like this:

show

TagInput support rightElement and has achieved it. So It will be very simple.

### PR DESCRIPTION
**Pull request by janewuwz**
#### Reviewers should focus on:
MultiSelect example

### CODE REVIEW

**Comment by palantirtech** [comment]
Thanks for your interest in [palantir/blueprint](https://github.com/palantir/blueprint), @janewuwz! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

File: packages/docs-app/src/examples/select-examples/multiSelectExample.tsx [hunk 1]
```diff
@@ -57,6 +57,13 @@ export class MultiSelectExample extends BaseExample<IMultiSelectExampleState> {
             undefined
         );
 
+        const clearButton =
+            films.length > 0 ? (
+                <Button className={Classes.MINIMAL} icon={"cross"} onClick={this.handleClear} />
```
Comment (giladgray) [inline packages/docs-app/src/examples/select-examples/multiSelectExample.tsx]: nit: `icon="cross"` (no braces)

File: packages/docs-app/src/examples/select-examples/multiSelectExample.tsx [hunk 2]
```diff
@@ -176,4 +183,8 @@ export class MultiSelectExample extends BaseExample<IMultiSelectExampleState> {
             this.setState({ [prop]: event.currentTarget.checked });
         };
     }
+
+    private handleClear = () => {
+        this.setState({ films: [] });
```
Comment (giladgray) [inline packages/docs-app/src/examples/select-examples/multiSelectExample.tsx]: nit: small fns like this work great on one line. 
`private name = () => this.setState(...)`

**Comment by janewuwz** [comment]
<h3>Follow code review</h3>

Preview: <strong><a href='https://13-126946346-gh.circle-artifacts.com/0/home/circleci/project/packages/docs-app/dist/index.html' target='_blank'>documentation</a> | <a href='https://13-126946346-gh.circle-artifacts.com/0/home/circleci/project/packages/landing-app/dist/index.html' target='_blank'>landing</a> | <a href='https://13-126946346-gh.circle-artifacts.com/0/home/circleci/project/packages/table-dev-app/dist/index.html' target='_blank'>table</a></strong>

**Comment by giladgray** [comment]
thanks @janewuwz!

**Pull request merged**